### PR TITLE
Fix typos in manual

### PR DESCRIPTION
**Pull request by m-holger**
Fix typos in cli.rst up to section "Page Ranges"

### CODE REVIEW

File: manual/cli.rst [hunk 1]
```diff
@@ -2715,7 +2711,7 @@ These options are valid between :qpdf:ref:`--add-attachment` and ``--``.
 
 .. qpdf:option:: --mimetype=type/subtype
 
-   .. help: attachment mime type, e.g. application/pdf
+   .. help: attachment mime type (e.g. application/pdf)
```
Comment (jberkenbilt) [inline manual/cli.rst]: Unfortunately, I had to change this to remove the trailing closed parentheses for a technical reason as explained in the "Note: 2022-01-22" comment at the top of cli.rst. I originally had it in parentheses.

**Review by jberkenbilt — COMMENTED** [review]
At a quick glance, these look good. Thanks. Rather than literally merging this, I'll pull the changes locally and incorporate them or tweak them as I see fit. Thanks again for your help. If I have some time, I may go ahead and accept changes up to this point including the ones you sent by email. It just depends on how much time I have for this and how it fits with my other train of thought since I'm starting to get into heavier refactoring work.

File: manual/cli.rst (hunk 1 above)
Comment (m-holger) [reply manual/cli.rst]: Sorry, missed that one.

Comment (jberkenbilt) [reply manual/cli.rst]: Yeah, I hate to have some technical detail interfere with better text, but it was the only case, and I decided it wasn't worth working around.

**Comment by jberkenbilt** [comment]
Changes incorporated locally. Thanks!

**Pull request closed without merging**